catalyst_voices/lib/pages/account/account_page.dart

```diff
@@ -0,0 +1,268 @@
+      ),
+      padding: const EdgeInsets.all(16),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Row(
+            children: [
+              Expanded(
+                child: Text(
+                  context.l10n.catalystKeychain,
+                  style: Theme.of(context).textTheme.titleLarge,
+                ),
+              ),
+              VoicesTextButton.custom(
+                leading: VoicesAssets.icons.x.buildIcon(),
+                color: Theme.of(context).colors.iconsError,
+                onTap: onRemoveKeychain,
+                child: Text(
+                  context.l10n.removeKeychain,
+                ),
+              ),
+            ],
+          ),
+          if (connectedWallet != null)
+            Padding(
+              padding: const EdgeInsets.symmetric(vertical: 24),
+              child: Text(
+                context.l10n.walletConnected,
+                style: Theme.of(context).textTheme.titleMedium,
+              ),
+            ),
+          if (connectedWallet != null)
+            Row(
+              children: [
+                VoicesIconButton.filled(
+                  style: ButtonStyle(
+                    padding: WidgetStateProperty.all(EdgeInsets.zero),
+                    foregroundColor: WidgetStateProperty.all(
+                      Theme.of(context).colors.successContainer,
+                    ),
+                    backgroundColor: WidgetStateProperty.all(
+                      Theme.of(context).colors.success,
+                    ),
+                  ),
+                  child: VoicesAssets.icons.check.buildIcon(),
+                ),
+                const SizedBox(width: 12),
+                Text(
+                  connectedWallet!,
+                  style: Theme.of(context).textTheme.bodyLarge,
+                ),
+              ],
+            ),
+          if (roles.isNotEmpty)
+            Padding(
+              padding: const EdgeInsets.only(
+                top: 40,
+                bottom: 24,
+              ),
+              child: Text(
+                context.l10n.currentRoleRegistrations,
+                style: Theme.of(context).textTheme.titleMedium,
+              ),
+            ),
+          if (roles.isNotEmpty)
+            Text(
+              roles
+                  .map((e) => _formatRoleBullet(e, defaultRole, context))
+                  .join('\n'),
+              style: Theme.of(context).textTheme.bodyLarge,
+            ),
+        ],
+      ),
+    );
+  }
+
+  String _formatRoleBullet(
+    AccountRole role,
+    AccountRole? defaultRole,
+    BuildContext context,
+  ) {
+    String label;
+    if (role == defaultRole) {
+      label = '${role.name(context)} (${context.l10n.defaultRole})';
+    } else {
+      label = role.name(context);
+    }
+    return ' • $label';
+  }
+}
+
+enum AccountRole {
+  voter,
+  proposer,
+  drep,
+}
```

Comment:
How about moving it to `models` package? Where **Spaces** and others live?

Reply:
Yeah, definitely, we should do, missed it  @digitalheartxs